lib/arguments/command.js

```diff
@@ -4,6 +4,7 @@ import {getStartTime} from '../return/duration.js';
 import {joinCommand} from './escape.js';
 import {normalizeFdSpecificOption} from './specific.js';
 
+// Compute `result.command`, `result.escapedCommand` and `verbose`-related information
```

Comment:
Maybe standardize with `.` at the end? Some comments have it and some don't.

Reply:
Currently the comments do not use a `.` when there is a single sentence, and use a `.` otherwise.

Would you prefer to always using a `.`, even when there is a single sentence?